leetcode/common_factors

```diff
@@ -0,0 +1,11 @@
+class Solution {
```

Comment:
Please add a small description of what the program is about and how it works.
You should name your file as per what the problem number is (e.g.: `2048.c`), as well as adding it to LeetCode's [`README.md`](https://github.com/TheAlgorithms/C/blob/master/leetcode/README.md).